### PR TITLE
returns httpInternalServerError if pdf fails when computing TOC

### PR DESCRIPTION
**Pull request by danduk82**
Add a missing handled exception

### CODE REVIEW

**Comment by pfirpfel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyramid_oereb/pyramid_oereb/pullRequest?prid=5591587&bid=18320297)

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 1]
```diff
@@ -121,26 +123,29 @@ def __call__(self, value, system):
...
+                    pdf.write(print_result.content)
+                    pdf_reader = PdfFileReader(pdf)
+                    x = []
+                    for i in range(len(pdf_reader.getOutlines())):
+                        x.append(pdf_reader.getOutlines()[i]['/Page']['/StructParents'])
+                    try:
+                        true_nb_of_toc = min(x)-1
+                    except ValueError:
+                        true_nb_of_toc = 1
+
```
Comment (pfirpfel) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py:138]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/pyramid_oereb/pyramid_oereb/pullRequest?prid=5591587)

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 2]
```diff
@@ -121,26 +123,29 @@ def __call__(self, value, system):
...
+                    if true_nb_of_toc != extract_as_dict['nbTocPages']:
+                        log.warning('nbTocPages in result pdf: {} are not equal to the one predicted : {}, request new pdf'.format(true_nb_of_toc,extract_as_dict['nbTocPages'])) # noqa
+                        extract_as_dict['nbTocPages'] = true_nb_of_toc
+                        print_result = requests.post(
+                            pdf_url,
+                            headers=pdf_headers,
+                            data=json.dumps(spec)
+                        )
+        except PdfReadError:
+             raise HTTPInternalServerError('a problem occurred while generating the pdf file') # noqa
```
Comment (jwkaltz) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py:148]: Shouldn't we log some information about the PdfReadError here?

Comment (danduk82) [reply pyramid_oereb/contrib/print_proxy/mapfish_print.py]: indeed

Comment (jwkaltz) [reply pyramid_oereb/contrib/print_proxy/mapfish_print.py]: I mean, the contents of the PdfReadError?

Comment (danduk82) [reply pyramid_oereb/contrib/print_proxy/mapfish_print.py]: done

**Pull request merged**